.buildconfig/ubuntu-latest.yml

```diff
@@ -4,37 +4,10 @@ channels:
 dependencies:
   - cmake
   - conan>=1.45.0
-  - confuse
   - cppcheck
-  - graphlib-backport
   - gxx_linux-64 11.1.*
-  - h5py
-  - hypothesis
-  - ipympl<0.9
-  - ipywidgets
-  - jinja2==3.0.3  # jupyter/nbconvert#1736
-  - markupsafe>=1.1.1,<2.1.0
-  - matplotlib-base
-  - nbconvert=6.4.5
-  - nbsphinx
   - ninja
-  - nomkl
-  - numpy>=1.20.0
-  - pandas
-  - pooch
-  - pytest
-  - pytest-xdist
-  - python-graphviz
-  - pythreejs
-  - pyyaml
-  - scipy>=1.7.0
-  - sphinx
-  - sphinx-book-theme==0.2  # 0.3 breaks custom launchbuttons
-  - sphinx-copybutton
+  - pandoc
```

Comment:
Why do we still need this here?

Reply:
`pandoc` is not available on PyPI. We could alternatively install it via `apt`?

Reply:
Don't think this would improve things. Just leave it.

Reply:
So you are installing a `conda` package that is later needed by a `sphinx` package that you get from `pip`? Why doesn't `sphinx` get this dependency itself, if it needs it?

Reply:
Because it cannot. `pandoc` has to be installed via `apt` or other (here: `conda`).

Reply:
Hmm, apparently it *is* on PyPI. Not sure why I remember it wasn't, or what the issue was. I will try.

Reply:
False: https://pypi.org/project/pandoc/ is not the pandoc we need. Reverted to installing with conda.